tests/acceptance/features/apiSharingNg/enableDisableShareSync.feature

```diff
@@ -413,3 +413,273 @@ Feature:  enable or disable sync of incoming shares
+            ],
+            "properties": {
+              "code" : {
+                "const": "invalidRequest"
+              },
+              "innererror" : {
+                "type": "object",
+                "required": [
+                  "date",
+                  "request-id"
+                ]
+              },
+              "message" : {
+                "const": "converting to drive items failed"
+              }
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: disable sync of shared resource from Personal Space when sherer is deleted
+    Given user "Brian" has disabled the auto-sync share
+    And user "Alice" has uploaded file with content "hello world" to "/textfile0.txt"
+    And we save it into "FILEID"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile0.txt |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Viewer        |
+    And the user "Admin" has deleted a user "Alice"
+    When user "Brian" tries to disable sync of share "textfile0.txt" using the Graph API
+    Then the HTTP status code should be "424"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["error"],
+        "properties": {
+          "error": {
+            "type": "object",
+            "required": [
+              "code",
+              "innererror",
+              "message"
+            ],
+            "properties": {
+              "code" : {
+                "const": "invalidRequest"
+              },
+              "innererror" : {
+                "type": "object",
+                "required": [
+                  "date",
+                  "request-id"
+                ]
+              },
+              "message" : {
+                "const": "unmounting share failed"
+              }
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: enable sync of shared resource from Project Space when auto-sync of sharee is disabled and sherer is deleted
+    Given user "Brian" has disabled the auto-sync share
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "NewSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "NewSpace" with content "hello world" to "/textfile0.txt"
+    And we save it into "FILEID"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile0.txt |
+      | space           | NewSpace      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Viewer        |
+    And the user "Admin" has deleted a user "Alice"
+    When user "Brian" enables share sync of a resource "<<FILEID>>" using the Graph API
```

Comment:
please, use this existing step
```suggestion
    When user "Brian" enables sync of share "textfile0.txt" offered by "Alice" from "NewSpace" space using the Graph API
```

Reply:
```
$itemId = $this->spacesContext->getResourceId($offeredBy, $space, $share);
```
Because we delete sharer, propfind request fails and thus this step was used so that it uses item id stored by previous step.